model/host/db.go

```diff
@@ -29,92 +29,97 @@ const (
+	HasContainersKey                       = bsonutil.MustHaveTag(Host{}, "HasContainers")
+	ParentIDKey                            = bsonutil.MustHaveTag(Host{}, "ParentID")
+	DockerOptionsKey                       = bsonutil.MustHaveTag(Host{}, "DockerOptions")
+	ContainerImagesKey                     = bsonutil.MustHaveTag(Host{}, "ContainerImages")
+	ContainerBuildAttemptKey               = bsonutil.MustHaveTag(Host{}, "ContainerBuildAttempt")
+	LastContainerFinishTimeKey             = bsonutil.MustHaveTag(Host{}, "LastContainerFinishTime")
+	SpawnOptionsKey                        = bsonutil.MustHaveTag(Host{}, "SpawnOptions")
+	ContainerPoolSettingsKey               = bsonutil.MustHaveTag(Host{}, "ContainerPoolSettings")
+	InstanceTagsKey                        = bsonutil.MustHaveTag(Host{}, "InstanceTags")
+	SSHKeyNamesKey                         = bsonutil.MustHaveTag(Host{}, "SSHKeyNames")
+	SSHPortKey                             = bsonutil.MustHaveTag(Host{}, "SSHPort")
+	HomeVolumeIDKey                        = bsonutil.MustHaveTag(Host{}, "HomeVolumeID")
+	PortBindingsKey                        = bsonutil.MustHaveTag(Host{}, "PortBindings")
+	IsVirtualWorkstationKey                = bsonutil.MustHaveTag(Host{}, "IsVirtualWorkstation")
+	SleepScheduleKey                       = bsonutil.MustHaveTag(Host{}, "SleepSchedule")
+	SpawnOptionsTaskIDKey                  = bsonutil.MustHaveTag(SpawnOptions{}, "TaskID")
+	SpawnOptionsTaskExecutionNumberKey     = bsonutil.MustHaveTag(SpawnOptions{}, "TaskExecutionNumber")
+	SpawnOptionsBuildIDKey                 = bsonutil.MustHaveTag(SpawnOptions{}, "BuildID")
+	SpawnOptionsTimeoutKey                 = bsonutil.MustHaveTag(SpawnOptions{}, "TimeoutTeardown")
+	SpawnOptionsSpawnedByTaskKey           = bsonutil.MustHaveTag(SpawnOptions{}, "SpawnedByTask")
+	VolumeIDKey                            = bsonutil.MustHaveTag(Volume{}, "ID")
+	VolumeDisplayNameKey                   = bsonutil.MustHaveTag(Volume{}, "DisplayName")
+	VolumeCreatedByKey                     = bsonutil.MustHaveTag(Volume{}, "CreatedBy")
+	VolumeTypeKey                          = bsonutil.MustHaveTag(Volume{}, "Type")
+	VolumeSizeKey                          = bsonutil.MustHaveTag(Volume{}, "Size")
+	VolumeExpirationKey                    = bsonutil.MustHaveTag(Volume{}, "Expiration")
+	VolumeNoExpirationKey                  = bsonutil.MustHaveTag(Volume{}, "NoExpiration")
+	VolumeHostKey                          = bsonutil.MustHaveTag(Volume{}, "Host")
+	VolumeMigratingKey                     = bsonutil.MustHaveTag(Volume{}, "Migrating")
+	VolumeAttachmentIDKey                  = bsonutil.MustHaveTag(VolumeAttachment{}, "VolumeID")
+	VolumeDeviceNameKey                    = bsonutil.MustHaveTag(VolumeAttachment{}, "DeviceName")
+	DockerOptionsStdinDataKey              = bsonutil.MustHaveTag(DockerOptions{}, "StdinData")
+	SleepScheduleNextStopTimeKey           = bsonutil.MustHaveTag(SleepScheduleInfo{}, "NextStopTime")
```

Comment:
Was this a weird merge? This seems unrelated to your changes.

Reply:
weird, I don't see this in the files tab, were you looking at the merge commit directly?

Reply:
It looks like it's part of [this merge ](https://github.com/evergreen-ci/evergreen/commit/a12320581a042adfa2ba3c0793ad2b14e4839110)

Reply:
I realized what was happening. When you filter commits to "since your last review", if there is a merge in there, that is shown as well.  
<img width="498" alt="image" src="https://github.com/evergreen-ci/evergreen/assets/13104717/eac78e5d-1665-41b3-8c89-5fe3253868de">

Reply:
Oh cool -- i very rarely use this setting so I didn't consider that. Thanks!